cernopendata/modules/fixtures/data/records/delphi-documentation-manuals.json

```diff
@@ -0,0 +1,205 @@
+[
+  {
+    "abstract": {
+      "description": "Detailed description of the DELPHI conditions database contents, and the tools coming with it."
+    },
+    "accelerator": "CERN-LEP",
+      "collaboration": {
+      "name": "DELPHI collaboration",
+      "recid": "27050"
+    },
+    "date_created": [
+      "1993"
+    ],
+    "date_published": "2024",
+    "distribution": {
+      "formats": [
+        "pdf"
+      ],
+      "number_files": 3
+    },
+    "experiment": [
+      "DELPHI"
+    ],
+    "files": [
+      {
+        "checksum": "adler32:18d0bebc",
+        "size": 404470,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/documentation/ddb/cargo600.pdf"
+      },
+      {
+        "checksum": "adler32:9ef43714",
+        "size": 229767,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/documentation/ddb/ddapp600.pdf"
+      },
+      {
+        "checksum": "adler32:63ea0076",
+        "size": 557435,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/documentation/ddb/dplot.pdf"
+      }
+    ],
+    "license": {
+      "attribution": "CC0"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "27001",
+    "title": "DELPHI database documentation",
+    "type": {
+      "primary": "Supplementaries",
+      "secondary": [
+        "DELPHI manuals"
+      ]
+    }
+  },
+  {
+    "abstract": {
+      "description": "DELPHI user analysis framework documentation, Skelana manuals."
+    },
+    "accelerator": "CERN-LEP",
+    "collaboration": {
+      "name": "DELPHI collaboration",
+      "recid": "27050"
+    },
+    "date_created": [
+      "1999"
+    ],
+    "date_published": "2024",
+    "distribution": {
+      "formats": [
+        "pdf"
+      ],
+      "number_files": 2
+    },
+    "experiment": [
+      "DELPHI"
+    ],
+    "files": [
+      {
+        "checksum": "adler32:9a26470d",
+        "size": 174717,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/documentation/skelana/99_175_prog_239.pdf"
+      },
+      {
+        "checksum": "adler32:4bac9f7a",
+        "size": 264402,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/documentation/skelana/skelana.pdf"
```

Comment:
When I see a file named "skelana.pdf", I consider it to be the canonical version... Whilst "99_175_prog_239" is named a bit opaquely and seems to provide later updates.

What about explaining this in the description?  Currently, the description does not provide much more information than the title:

![skelana](https://github.com/user-attachments/assets/a11d948e-ae9d-47e6-aabc-c331ca5b0988)


It may be good to expand the description phase to say something like: "The Skelana analysis framework version 1 is documented in the file `skelana.pdf`. The later updates for Skelana version 2 are described in file 99_...".

... or perhaps consider renaming files?

Reply:
I kept the names are they are since these are consistent with the manuals elsewhere. It's now described in the abstract, as suggested.